spec/double_entry/reporting/line_aggregate_filter_spec.rb

```diff
@@ -78,8 +85,31 @@
       end
     end
 
-    context 'with no code specified' do
+    context 'with a partner_account specified' do
```

Comment:
Yeah good point, it wouldn't hurt to make it a little clearer!  We do already have a case for your third point (it's above the one you commented on :) )